Source/WebKit/WebProcess/WebPage/UnifiedTextReplacementController.h

```diff
@@ -109,6 +109,7 @@ class UnifiedTextReplacementController final {
 
     using TextIndicatorCharacterRange = std::pair<WTF::UUID, WebCore::CharacterRange>;
     Vector<std::pair<WTF::UUID, Vector<TextIndicatorCharacterRange>>> m_textIndicatorCharacterRangesForSessions;
+    Vector<std::pair<WTF::UUID, std::pair<WTF::UUID, WebCore::SimpleRange>>> m_remainingRangesForSessions;
```

Comment:
can this be a `HashMap<UUID, HashMap<UUID, SimpleRange>>`?

Reply:
Not today.